src/docker.ts

```diff
@@ -214,15 +222,15 @@ export async function setupContainer(
     if (id) {
       console.log(chalk.blue(`cleanup: removing ${type} container`));
       const c = docker.getContainer(id);
-      await c.remove({ force: true });
+      await c.stop({ t: 5 });
```

Comment:
is t=5 the grace period?

Reply:
Yeah docker will send a SIGTERM immediately, wait 5 seconds, then send a SIGKILL: https://docs.docker.com/reference/cli/docker/container/stop/